diagnostic_updater/include/diagnostic_updater/DiagnosticStatusWrapper.hpp

```diff
@@ -164,10 +166,11 @@ class DiagnosticStatusWrapper : public diagnostic_msgs::DiagnosticStatus
   void summaryf(unsigned char lvl, const char * format, ...)
   {
     va_list va;
-    char buff[1000];     // @todo This could be done more elegantly.
+    char buff[1000];  // @todo This could be done more elegantly.
     va_start(va, format);
-    if (vsnprintf(buff, 1000, format, va) >= 1000) {
-      ROS_DEBUG("Really long string in DiagnosticStatusWrapper::addf, it was truncated.");
+    if (vsnprintf(buff, sizeof(buff), format, va) >= 1000) {
```

Comment:
If this is going to change, it can be ` if (vsnprintf(buff, sizeof(buff), format, va) >= sizeof(buff) {`